include/libtorrent/torrent_peer.hpp

```diff
@@ -276,4 +276,3 @@ namespace libtorrent {
 }
 
 #endif
-
```

Comment:
Don't remove newline at the end of file, git won't like this.
I would recommend reading [Newlines and Version Control](https://mrkiffie.com/2016/newlines-and-version-control/)

Reply:
heh, it also used to be undefined behavior, pre C++11 :)

Reply:
I only remove _extra_ newlines at the end

Reply:
Line 279 is not an `extra` one. If it was followed by another empty(new) line then it could be removed. In other words if the file ends with a "\n\n" or "\r\n\r\n" then it has no extra new lines.

Reply:
I'm not sure I understand  what are you talking about

Reply:
traditionally, every line was required to end with a '\n' character in C and C++

Reply:
for future reference, when doing github web based code review, the web interface give you a nice hint of that missing newline at the end
![screen shot 2018-01-27 at 11 02 10 am](https://user-images.githubusercontent.com/143813/35473685-aee9de22-0351-11e8-96f2-9b62930bc883.png)